### PR TITLE
Getting logs fixed

### LINKED ISSUE #815: Get server logs verbs
#### Description of issue
@Tsurupeta said he can fix the log verbs, so why not?

#### Difference between expected and actual behavior
in this time logs didnt work

#### Steps to reproduce
-Get admins permissions?

_Фикс для мужичков пыж-пыж. Починить get server logs вёрбы и иные_

### PR DESCRIPTION
**Pull request by Tsurupeta**
#815 

## Changelog

:cl:
admin: Added ability to view server logs from the game.
/:cl:

### CODE REVIEW

File: code/_helpers/files.dm [hunk 1]
```diff
@@ -32,3 +32,68 @@
...
+		return
+
+	return path
+
+#define FTPDELAY 200	//200 tick delay to discourage spam
+/*	This proc is a failsafe to prevent spamming of file requests.
+	It is just a timer that only permits a download every [FTPDELAY] ticks.
+	This can be changed by modifying FTPDELAY's value above.
+
+	PLEASE USE RESPONSIBLY, Some log files canr each sizes of 4MB!	*/
```
Comment (MACIEKBAKI) [inline code/_helpers/files.dm]: this spelling mistake will annoy me greatly

Comment (MACIEKBAKI) [reply code/_helpers/files.dm]: (not really but hey)

**Comment by MACIEKBAKI** [comment]
where'd you yoink this from

**Comment by Tsurupeta** [comment]
> where'd you yoink this from

from original bay repository, we for some reason had this file unchecked and procs missing.

**Comment by MACIEKBAKI** [comment]
fix your checks,  there's some indentation stuff

**Comment by Tsurupeta** [comment]
> fix your checks, there's some indentation stuff

Just did that

**Comment by MACIEKBAKI** [comment]
based speedmerge

**Review by hry-gh — APPROVED** [review]

**Pull request merged**